src/error/GraphQLError.js

```diff
@@ -101,7 +101,8 @@ export function GraphQLError( // eslint-disable-line no-redeclare
 
   let _positions = positions;
   if (!_positions && nodes) {
-    _positions = nodes.map(node => node.loc && node.loc.start).filter(Boolean);
+    _positions = nodes.map(node => node.loc && node.loc.start)
```

Comment:
alternatively this could be `nodes.filter(node => node.loc != null).map(node => node.loc.start)` though Flow may complain about that.